src/validation.ts

```diff
@@ -0,0 +1,20 @@
+import { ValidationRules } from "aurelia-validation";
```

Comment:
"src" is not an appropriate location.   I recommend a `ValidationService.ts` in services.

Reply:
it's not really a service. Is just a file with custom validation rules. Is there a place for such things?

Reply:
I would put it where I suggested.  I think calling it a service is ok.